lib/tasks/mastodon.rake

```diff
@@ -82,6 +82,22 @@ namespace :mastodon do
     end
   end
 
+  namespace :settings do
+    desc 'Open registrations on this instance'
+    task open_registrations: :environment do
+      setting = Setting.where(var: 'open_registrations').first
+      setting.value = 'true'
```

Comment:
ah, mistake on my part, appears to have been working anyway because 'true' is truthy. Pushed a fix for consistency.